cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -221,6 +235,21 @@ public void deleteNamespace(final Id.Namespace namespaceId)
     LOG.info("All data for namespace '{}' deleted.", namespaceId);
   }
 
+  private void deleteAllSchedules(Id.Namespace namespaceId) throws NotFoundException, SchedulerException {
```

Comment:
Moved the method to the `Schedule` interface and added its implementation in the classes that inherits it.